packages/datepickers/src/elements/DatePickerRange/components/Month.tsx

```diff
@@ -121,11 +121,11 @@ export const Month = forwardRef<HTMLDivElement, IMonthProps>(
 
       if (isPreviousMonth) {
         return (
-          <StyledCalendarItem key={`day-${itemsIndex}`} isCompact={isCompact}>
+          <StyledCalendarItem key={`day-${itemsIndex}`} $isCompact={isCompact}>
             <StyledDay
-              isCompact={isCompact!}
-              isPreviousMonth
-              isDisabled
+              $isCompact={isCompact!}
+              $isPreviousMonth
+              aria-disabled
```

Comment:
Does this render `aria-disabled="true"` as expected?

Reply:
It does. I tested the result in Storybook, and also see these propagate in tests as expected. If being more explicit is preferred (`"true"` / `"false"` / `undefined`), we can do that instead though.